components/cdc/src/service.rs

```diff
@@ -248,16 +254,27 @@ impl EventFeedHeaders {
 pub struct Service {
     scheduler: Scheduler<Task>,
     memory_quota: Arc<MemoryQuota>,
+    pool: Arc<Runtime>,
 }
 
 impl Service {
     /// Create a ChangeData service.
     ///
     /// It requires a scheduler of an `Endpoint` in order to schedule tasks.
     pub fn new(scheduler: Scheduler<Task>, memory_quota: Arc<MemoryQuota>) -> Service {
+        let pool = Arc::new(
+            tokio::runtime::Builder::new_multi_thread()
+                .worker_threads(1)
+                .thread_name("cdc-watchdog")
+                .enable_all()
+                .build()
+                .unwrap(),
+        );
```

Comment:
This seems quite similar to using `ctx.spawn`, especially since it's a multi-thread runtime with only one worker thread.

Reply:
I wrote it this way to make all Streams share the same pool, thereby reducing the number of threads.